clients/python/play.py

```diff
@@ -21,9 +20,8 @@ def main():
 
         # TODO call player AI here
         print(str(game))
-        api.send_actions(game_id,
-                         [CellActions(game.me.cells[0].id,
-                          Vec2(0, 0), False, False, False, 0)])
+
+        api.send_actions(game_id, [cell.actions() for cell in game.me.cells])
```

Comment:
I guess. I didn't want to get into testing with a fake API etc, but I guess I'll do it. I'll update the PR today.